Modules/Scripted/ScreenCapture/ScreenCapture.py

```diff
@@ -671,16 +682,23 @@ def onCaptureButton(self):
     slicer.app.setOverrideCursor(qt.Qt.WaitCursor)
     captureAllViews = self.captureAllViewsCheckBox.checked
     transparentBackground = self.transparentBackgroundCheckBox.checked
+    showViewControllers = self.showViewControllersCheckBox.checked
     if captureAllViews:
-      self.logic.showViewControllers(False)
+      self.logic.showViewControllers(showViewControllers)
+    elif showViewControllers:
+      logging.warning("View controllers are only available to be shown when capturing all views.")
     try:
       if numberOfSteps < 2:
         if imageFileNamePattern != self.snapshotFileNamePattern or outputDir != self.snapshotOutputDir:
           self.snapshotIndex = 0
-        [filename, self.snapshotIndex] = self.logic.getNextAvailableFileName(outputDir, imageFileNamePattern, self.snapshotIndex)
+        if outputDir:
+          [filename, self.snapshotIndex] = self.logic.getNextAvailableFileName(outputDir, imageFileNamePattern, self.snapshotIndex)
+        else:
+          filename = None
         view = None if captureAllViews else self.logic.viewFromNode(viewNode)
-        self.logic.captureImageFromView(view, filename, transparentBackground)
-        self.logic.addLog("Write "+filename)
+        self.logic.captureImageFromView(view, filename, transparentBackground, volumeNode=self.volumeNodeComboBox.currentNode())
```

Comment:
Use volumeNode=None if numberOfSteps>1 (otherwise we would keep writing images into the same node). Or, implement writing the output into a sequence node (you could create a sequence node and browser node and use the chosen volume node as proxy node).